stable-25-1-3 + stable-24-4: ydb/tests/compatibility/test_compatibility.py.TestCompatibility.test_tpch1[restart_stable-25-1-3_to_stable-24-4-column]

https://github.com/ydb-platform/ydb/actions/runs/16421833905

https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/16421833905/ya-main-x86-64/try_1/ya-test.html#MUTE

Тест 
ydb/tests/compatibility/test_compatibility.py.TestCompatibility.test_tpch1[restart_stable-25-1-3_to_stable-24-4-column] 

Во время работы 25-1-3 + 24-4

```
E   VERIFY failed (2025-07-21T16:25:27.509407Z): tablet_id=72075186224037948;parent_id=[1:7529572139271917616:101];verification=seqDataColumnIds.erase(i);fline=general_compaction.cpp:123;id=4294967040;
E   ydb/library/actors/core/log.cpp:747
E   ~TVerifyFormattedRecordWriter(): requirement false failed
E   0. /-S/util/system/yassert.cpp:83: NPrivate::InternalPanicImpl @ 0xAA5D67C
E   1. /-S/util/system/yassert.cpp:55: NPrivate::Panic @ 0xAA5887B
E   2. /-S/ydb/library/actors/core/log.cpp:747: NActors::TVerifyFormattedRecordWriter::~TVerifyFormattedRecordWriter @ 0xB5AEE14
E   3. /-S/ydb/core/tx/columnshard/engines/changes/general_compaction.cpp:123: NKikimr::NOlap::NCompaction::TGeneralCompactColumnEngineChanges::BuildAppendedPortionsByChunks @ 0x12450844
E   4. /-S/ydb/core/tx/columnshard/engines/changes/general_compaction.cpp:173: NKikimr::NOlap::NCompaction::TGeneralCompactColumnEngineChanges::DoConstructBlobs @ 0x12451F9E
E   5. /-S/ydb/core/tx/columnshard/engines/changes/abstract/abstract.cpp:23: NKikimr::NOlap::TColumnEngineChanges::ConstructBlobs @ 0x12444B51
E   6. /-S/ydb/core/tx/columnshard/columnshard_impl.cpp:534: NKikimr::NColumnShard::TChangesTask::DoExecute @ 0x126DA228
E   7. /-S/ydb/core/tx/conveyor/usage/abstract.cpp:13: NKikimr::NConveyor::ITask::Execute @ 0x123E6A6F
E   8. /-S/ydb/core/tx/conveyor/service/worker.cpp:10: NKikimr::NConveyor::TWorker::ExecuteTask @ 0x18E200AD
E   9. /-S/ydb/library/actors/core/executor_thread.cpp:251: NActors::TGenericExecutorThread::Execute<NActors::TMailboxTable::THTSwapMailbox> @ 0xB591C98
E   10. /-S/ydb/library/actors/core/executor_thread.cpp:440: NActors::TGenericExecutorThread::ProcessExecutorPool(NActors::IExecutorPool *)::(anonymous class)::operator() @ 0xB589BB1
E   11. /-S/ydb/library/actors/core/executor_thread.cpp:493: NActors::TGenericExecutorThread::ProcessExecutorPool @ 0xB5895A9
E   12. /-S/ydb/library/actors/core/executor_thread.cpp:524: NActors::TExecutorThread::ThreadProc @ 0xB58A346
E   13. /-S/util/system/thread.cpp:244: (anonymous namespace)::TPosixThread::ThreadProxy @ 0xAA618B9
E   14. ??:0: ?? @ 0x7F693879DAC2
E   15. ??:0: ?? @ 0x7F693882F84F
E   Daemon failed with message: Unexpectedly finished before stop.
```

--- a/ydb/tests/compatibility/test_compatibility.py
+++ b/ydb/tests/compatibility/test_compatibility.py
@@ -97,6 +97,9 @@ class TestCompatibility(RestartToAnotherVersionFixture):
         if date64 and min(self.versions) < (25, 1):
             pytest.skip("date64 is not supported in 24-4")
 
+        if store_type == "column" and min(self.versions) < (25, 1) and max(self.versions) >= (25, 1):
+            pytest.skip("no compatibility with 24-4")
+
         if date64:
             date_args = ["--datetime-types=dt64"]
         else:

